source/Government.cpp

```diff
@@ -99,11 +101,14 @@ void Government::Load(const DataNode &node)
 						penaltyFor[ShipEvent::CAPTURE] = grand.Value(1);
 					else if(grand.Token(0) == "destroy")
 						penaltyFor[ShipEvent::DESTROY] = grand.Value(1);
+					else if(grand.Token(0) == "scanning")
+						penaltyFor[ShipEvent::SCAN_OUTFITS] = grand.Value(1);
```

Comment:
You should also set `penaltyFor[ShipEvent::SCAN_CARGO]` since `scanning` applies to both.